src/core/ext/transport/inproc/inproc_transport.cc

```diff
@@ -832,6 +847,9 @@ void op_state_machine_locked(inproc_stream* s, grpc_error* error) {
         "op_state_machine scheduling recv-trailing-md-on-complete");
     s->trailing_md_recvd = true;
     s->recv_trailing_md_op = nullptr;
+    // Since we are only pretending to have received the trailing MD, it would
+    // be ok (not an error) if the client-side sent it also.
+    s->trailing_md_recvd_implicit_only = true;
```

Comment:
I got slightly confused reading the comment "if the client-side sent it also". Suggest changing it to - "if the client-side actually sends it later"

Reply:
Done.